mro/__init__.py

```diff
@@ -1,23 +1,31 @@
-﻿
-import mro.connection
+﻿import mro.connection
 import mro.data_types
 import mro.table
 import mro.sqlite
+import mro.custom_types
 
 
 def disconnect():
     mro.connection.disconnect()
 
 
-def load_database(connection_function, exclude_tables=[]):
-
+def load_database(connection_function, hooks=None):
+    print("***********INITIALISING DATABASE************")
     mro.connection.set_connection_function(connection_function)
+    mro.connection.set_on_reconnect(init_db)
+    mro.connection.set_hooks(hooks)
```

Comment:
Added hooks here, as we used to register hooks with mro objects inside the_kore, however when the db moves and the db needs to reinitialise itself these objects no longer exist, so i moved the ability to add hooks to the db so it can re-register them when it reconnects